src/components/Dropdown/Dropdown.stories.jsx

```diff
@@ -229,4 +230,106 @@ storiesOf('Components/Dropdown', module)
                 />
             </div>
         );
+    })
+    .add('With Item Description', () => {
+        const [currentOption, setCurrentOption] = useState({label: 'Select something', value: null});
+        const handleOnChange = (e, item) => {
+            setCurrentOption(item);
+            action('onChange');
+            return true;
+        };
+
+        return (
+            <div style={{transform: 'scale(1)', height: '100vh', padding: '90px'}}>
+                <Dropdown
+                    icon={<Love/>}
+                    label={currentOption.label}
+                    value={currentOption.value}
+                    size={select('Size', DropdownSizes, DropdownSizes.Small)}
+                    isDisabled={boolean('Disabled', false)}
+                    maxWidth={text('Max width', '320px')}
+                    data={[
```

Comment:
A suggestion is to simplify the story and mutualize the data (even if I don't know if we will reuse it). You can create a new data file in `src/data`.
You have a data file per data object.

Reply:
Done